web/styles/compose.css

```diff
@@ -774,6 +771,66 @@ a.compose_control_button.hide {
     margin: auto;
 }
 
+#compose_stream_selection_dropdown {
+    display: flex;
+    justify-content: flex-start;
+    flex: 0 1 200px;
+    min-width: 0;
+
+    .stream_header_colorblock {
+        margin: 0;
+    }
+
+    .dropdown-toggle {
+        border-radius: 0 4px 4px 0 !important;
+        display: flex;
+        min-width: 0;
+    }
+
+    .dropup .dropdown-menu {
+        top: auto;
+        bottom: 100%;
+        left: -10px;
+        margin-bottom: 17px;
+    }
+
+    .dropdown .dropdown-menu {
+        top: auto;
+        left: -10px;
+    }
+
+    #compose_select_stream_name {
+        flex-grow: 1;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+    }
+
+    .dropdown-list-body .list_item a {
+        white-space: normal;
+    }
+
+    .fa-chevron-down {
+        float: right;
+        padding-bottom: 2px;
+        padding-left: 5px;
+        color: hsl(0deg 0% 58%);
+        font-weight: lighter;
+    }
+
+    .text-warning {
+        color: inherit; /* TODO(compose-dropdown): do we want grey or something here? */
```

Comment:
I'm not sure how this is displayed, but is this still a TODO we need to resolve?

Reply:
It's the text color of "Select a stream" here:

<img width="895" alt="image" src="https://user-images.githubusercontent.com/5634097/229023719-760ba06d-2390-4407-aca0-73cdb8e6d307.png">

@alya @terpimost do you have opinions about what color to do here?

Reply:
The grey seems fine, and since we're getting rid of that bar in the redesign anyway, I think we can reconsider what it looks like when there's no stream selected in the new context.

Reply:
Yeah, that all sounds correct to me.

Reply:
Removed the comment